app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,92 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.errors.ProductNotFoundException;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+/**
+ * 상품을 관리합니다.
+ */
+@Service
+@Transactional
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    /**
+     * 주어진 의존성 객체를 사용하여 ProductService 객체를 생성합니다.
+     * @param productRepository 상품 repository
+     */
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 저장하고 있는 상품 목록을 반환합니다.
+     * @return 상품 목록
+     */
```

Comment:
아주 잘 작성하셨어요. 좀 더 심플하게 이렇게 해도 충분해 보입니다.
```suggestion
    /**
     * 상품 목록을 반환합니다.
     * @return 상품 목록
     */
```